src/YouTube.js

```diff
@@ -76,16 +110,29 @@ const YouTubeComponent = ({ url, ...props }) => {
         setReady(true);
     };
 
+    const clickButton = useCallback(() => {
+        playerRef.current.internalPlayer.playVideo();
+        setPlay(true);
+    }, []);
+
+    const playVideo = useCallback(() => {
+        setPlay(true);
+    }, []);
+    const pauseVideo = useCallback(() => {
+        setPlay(false);
+    }, []);
+
     return (
         <Box
             padding-top={videoId ? '56.25%' : '0'}
             min-height={videoId && '0'}
             height={videoId && '0'}
-            transition="opacity 0s initial .5s"
-            opacity={isReady ? '1' : '0'}
             {...rest}
         >
-            <Box {...override('YouTube Content')} display={!videoId && 'none'}>
+            <Box
+                {...override('YouTube Content')}
+                display={(!videoId || !isReady) && 'none'}
```

Comment:
It's better to use ternary operator here, because you'll pass `false` to css property

`(!videoId || !isReady) ? undefined : 'none'`